Exiled.Events/Patches/Events/Scp330/Eating.cs

```diff
@@ -0,0 +1,77 @@
+// -----------------------------------------------------------------------
+// <copyright file="Eating.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Patches.Events.Scp330
+{
+    #pragma warning disable SA1118
+    using System.Collections.Generic;
+    using System.Reflection;
+    using System.Reflection.Emit;
+
+    using Exiled.API.Features;
+    using Exiled.Events.EventArgs;
+
+    using HarmonyLib;
+
+    using InventorySystem.Items.Usables.Scp330;
+
+    using NorthwoodLib.Pools;
+
+    using static HarmonyLib.AccessTools;
+
+    /// <summary>
+    /// Patches <see cref="Scp330Bag.ServerOnUsingCompleted"/>.
+    /// Adds the <see cref="Handlers.Scp330.OnEating"/> event.
+    /// </summary>
+    [HarmonyPatch(typeof(Scp330Bag), nameof(Scp330Bag.ServerOnUsingCompleted))]
+    internal static class Eating
+    {
+        private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> instructions, ILGenerator generator)
+        {
+            List<CodeInstruction> newInstructions = ListPool<CodeInstruction>.Shared.Rent(instructions);
+
+            const int offset = -3;
+
+            var index = newInstructions.FindIndex(instruction => instruction.opcode == OpCodes.Callvirt && (MethodInfo)instruction.operand == Method(typeof(ICandy), nameof(ICandy.ApplyEffects))) + offset;
+
+            var returnLabel = generator.DefineLabel();
+
+            newInstructions.InsertRange(index, new[]
+            {
+                // Player.Get
```

Comment:
If you want to comment your own instructions, comment everything.
IE `Player.Get(args)`